content/en/blog/_posts/2024-06-21-consistent-read-from-cache.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: blog
+title: 'Kubernetes Consistent Reads from Cache: A Leap in Performance and Scalability'
+date: 2024-06-21
+slug: consistent-read-from-cache-beta
+---
+
+**Author:** Marek Siarkowicz (Google)
```

Comment:
We should incorporate the new mechanism for adding authors in the front-matter of the blog. Refer below suggestion.

```suggestion
slug: consistent-read-from-cache-beta
author: >
  Marek Siarkowicz (Google)
---
```